### PR TITLE
[fix] Removed old bash scripts and charts.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Clean up, remove cruft

### CODE REVIEW

File: kube-config [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```
Comment (jimmykarily) [inline kube-config:1]: Added by mistake?

File: quark-secrets-log.txt [hunk 2]
```diff
@@ -0,0 +1,275 @@
+2021-02-04T13:37:45.405Z	[34mINFO[0m	cmd/kubeconfig.go:16	Using in-cluster kube config
```
Comment (jimmykarily) [inline quark-secrets-log.txt:1]: also mistake?

File: kube-config (hunk 1 above)
Comment (andreas-kupries) [reply kube-config:1]: Likely yes. Will remove.

File: quark-secrets-log.txt (hunk 2 above)
Comment (andreas-kupries) [reply quark-secrets-log.txt:1]: Very likely yes. Will remove.

It seems that I was in too much of a hurry when I made this :(

File: kube-config (hunk 1 above)
Comment (andreas-kupries) [reply kube-config]: Done with commit 0d380cfe9c5b24cdbc7eec2615976d3c88af3723 (rebase).

File: quark-secrets-log.txt (hunk 2 above)
Comment (andreas-kupries) [reply quark-secrets-log.txt]: Done with commit 0d380cfe9c5b24cdbc7eec2615976d3c88af3723 (rebase).

**Comment by jimmykarily** [comment]
We should probably update these make targets now: https://github.com/SUSE/carrier/blob/main/cli/Makefile#L44-L59

no files should be copied over anymore, they become first class citizens. The only exception is the registry which we have to `helm package`

**Comment by jimmykarily** [comment]
This is a good first step. Next one would be to move the cli code outside the `cli` directory and make the root of the repo look like a go application project (so the contents of the `cli` directory move at the top level). Let's do it in steps. We can keep this PR small and merge it as it is. Maybe just update the Make targets for now.

**Comment by andreas-kupries** [comment]
Ah. These targets pull files from the directories I removed. These files have to be restored and placed into the embedded-files hierarchy. Ok. Will be done ...

**Comment by andreas-kupries** [comment]
Done with commit 8c0ce5a72fc6bf6536609fd5eb068f029144bb7b.
Files are already in the destination, so just removed the `cp` commands.

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**